### ISSUE TITLE
SynchronousOnlyOperation raised by Django test cases

### ISSUE THREAD

**Issue by m6312** [wkeeling/selenium-wire#358]
### Issue

Django `LiveServerTestCase` tests fail with `selenium-wire==4.3.3` when the tests try to access the database. These failures were introduced by `selenium-wire==4.3.3`, the tests worked with `selenium-wire==4.3.2`. I'm not familiar with `asyncio`, but [Django's ORM code doesn't currently have asynchronous support](https://docs.djangoproject.com/en/3.2/topics/async/#asynchronous-support) - perhaps that is part of the problem.

### Reproduction

Example test case. `test_foo` just makes a trivial database operation.

```
from django.contrib.auth import get_user_model
from django.test import LiveServerTestCase
from selenium.webdriver import ChromeOptions
from seleniumwire.webdriver import Chrome

class MyTestCase(LiveServerTestCase):
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        options = ChromeOptions()
        options.headless = True
        driver = Chrome(
            executable_path='/home/m6312/.local/bin/chromedriver',
            options=options,
            service_args=['--verbose', '--readable-timestamp'],
        )

    def test_foo(self):
        self.assertTrue(get_user_model().objects.all())
```

This is the traceback produced from running `./manage.py test <path to MyTestCase>`.

```
======================================================================
ERROR: test_foo (tests.test_foo.MyTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/m6312/my_project/tests/test_foo.py", line 20, in test_foo
    self.assertTrue(get_user_model().objects.all())
  File "/usr/lib/python3.6/unittest/case.py", line 680, in assertTrue
    if not expr:
  File "/home/m6312/my_venv/lib/python3.6/site-packages/django/db/models/query.py", line 284, in __bool__
    self._fetch_all()
  File "/home/m6312/my_venv/lib/python3.6/site-packages/django/db/models/query.py", line 1324, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/m6312/my_venv/lib/python3.6/site-packages/django/db/models/query.py", line 51, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/home/m6312/my_venv/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1173, in execute_sql
    cursor = self.connection.cursor()
  File "/home/m6312/my_venv/lib/python3.6/site-packages/django/utils/asyncio.py", line 24, in inner
    raise SynchronousOnlyOperation(message)
django.core.exceptions.SynchronousOnlyOperation: You cannot call this from an async context - use a thread or sync_to_async.
```

### Packages/versions

- Ubuntu 20.04.2
- PostgreSQL 13.2
- Python 3.6.13
- django==3.2.5
- selenium==3.141.0
- selenium-wire==4.3.3

**Comment by wkeeling** [comment]
Many thanks for the bug report. This issue must have been introduced with the recent threading fixes in 4.3.3. I'll investigate and try and get a fix in as soon as I can.

**Comment by wkeeling** [comment]
Fixed in v4.4.1
